### ISSUE TITLE
Integrate full-feature blog platform or RSS feed into Indico

### ISSUE THREAD

**Issue by glunardi** [indico/indico#3761]
In 2018, the Linux Plumbers Conference successfully adopted Indico for its call for papers, scheduling and sessions management. Many thanks to the entire Indico community again.

We used Wordpress in conjunction with Indico to ease the creation of web content for our audience. Content like:
- Announcing new tracks 
- Announcing new sessions
- Announcing changes to the call for abstracts for specific sessions and tracks
- News about co-located events 
- New about the location and evening events
- etc.

One of the main reason we use a traditional blog platform is a native RSS feed for all created web content (using Wordpress to syndicate to various planets/etc.).

Having the ability for every "page" and changes to show up into a RSS feed is beneficial to our audience.

Right now, we are creating some content using the Indico > Menu > Create Page way of creating content to some extent but Wordpress is still preferred by the committee.

In summary: either having an integrated RSS feed for Indico's web content would be great, or an integration with a full-feature blogging platform.

**Comment by ThiefMaster** [comment]
I think this is a bit of a very special usecase that's outside the normal scope of Indico.

However, it sounds like something that could be done "easily" (unless you want to replicate something like Wordpress ;)) as a plugin - regardless of whether it'd be event-scoped or global.

For a feed of updates you could use the signals Indico triggers on various events (and contribute missing ones via Pull Requests if needed) and then log them in a new table which you use to generate a RSS feed. For manual announcements you'd have to add a web interface to add entries there, but that's also pretty straightforward with WTForms. Or, if this thing is global, you could use the built-in news functionality and integrate that with your event (and/or a RSS feed).

Anyway, I think this discussion would be better suited on [our forum](https://talk.getindico.io/) since it's certainly an interesting topic and we can help you with tips on how to best do things with Indico, but not really within the scope of the application itself.

But if someone were to create a complete plugin that adds support for enabling a blog/newsfeed-like page in an event, I think that'd be a very useful feature for others as well. I'm sure there are more conferences who would benefit from this!

**Comment by glunardi** [comment]
Thank you for the prompt reply and already very insightful details.

Happy to read that there could be interest in such a feature and will create an entry on the forum for further discussion.
